lambda/dataImporter/src/lib/getProtectedAssets.js

```diff
@@ -0,0 +1,57 @@
+const axios = require("axios");
+const awsClient = require("aws-sdk");
+const { Buffer } = require("buffer");
+
+const getProtectedAssets = async (searchAfter = [], pageSize = 1000) => {
+  const cloudGuardApiEndpoint = process.env.CLOUDGUARD_API_ENDPOINT;
+  const cloudGuardApiKeysParameterStore =
+    process.env.CLOUDGUARD_API_KEYS_PARAMETER_STORE;
+  const cloudGuardApiPageSize = process.env.CLOUDGUARD_PAGE_SIZE;
+
+  const ssmClient = new awsClient.SSM({
+    apiVersion: "2014-11-06",
+    region: "ca-central-1",
```

Comment:
We always use ca-central-1 but it would be better to set this as an env var.

Reply:
node: `    region: process.env.AWS_API_REGION,`
terraform lambda resource: `      "AWS_API_REGION"                      = var.aws_region`

I think it makes sens to use the api in the deployed region ?